browsers/opera.json

```diff
@@ -499,6 +499,20 @@
           "status": "current",
           "engine": "Blink",
           "engine_version": "76"
+        },
+        "64": {
+          "release_date": "2019-08-21",
+          "release_notes": "https://blogs.opera.com/desktop/2019/09/opera-64-0-3417-11-beta-update/",
```

Comment:
Please don't add release notes and release dates for beta or nightly releases. 
This info is only useful for actual final releases.